fvm/evm/testutils/backend.go

```diff
@@ -221,7 +221,7 @@ type TestValueStore struct {
 	GetValueFunc             func(owner, key []byte) ([]byte, error)
 	SetValueFunc             func(owner, key, value []byte) error
 	ValueExistsFunc          func(owner, key []byte) (bool, error)
-	AllocateStorageIndexFunc func(owner []byte) (atree.StorageIndex, error)
+	AllocateStorageIndexFunc func(owner []byte) (atree.SlabIndex, error)
```

Comment:
We should maybe change the remaining "storage index" naming to "slab index"

Reply:
Improved this a bit in [8e4c7dc](https://github.com/onflow/flow-go/pull/6275/commits/8e4c7dcab0c76b1385483c2648da047a4b975133)